docs/guide/contributing.rst

```diff
@@ -31,37 +27,52 @@ We especially need contributions in these areas:
 * Developer tools or docs-as-code toolchains and workflows.
 * High-level discussions of common tools. 
 
-For help finding where a new set of topics best fit into the rest of 
-the guide, `ask in Slack`_, or `file a GitHub issue`_.
+For help organizing a new topic within the current guide, `ask in Slack`_, or `file a GitHub issue`_.
+
+Contribution guidelines
+-------------------------
+
+* Use a friendly and encouraging tone. It's a good way to write docs, so practice with docs about docs!
+
+* Attribute links and mention why linked resources are useful.
 
-Guidelines
------------
+* Use `reStructuredText (.rst)`_ (`Markdown`_ is also acceptable).
 
-Follow these guidelines for discussing tools: 
+* Focus content on general principles and best practices. Arguing over minor points impedes clarity. 
+
+* Avoid enforcing personal preferences. For example, if you recommend a word choice, tell the audience why it matters.
+
+Additionally, follow these guidelines when discussing tools: 
 
 * Ensure your content is of interest to a general audience.
 * Avoid advocacy and plugs about your favorite toolchain, even if it's open source.
 * Present specific use cases, how problems were solved, and what worked or didn't work well. 
-* Consider to first `file a GitHub issue`_ or contact one of the guide editors at guide@writethedocs.org.
+* Consider to first `file a GitHub issue`_ or contact a guide editor at guide@writethedocs.org.
+
+.. _update-guide-via-github:
+
+Editing a single guide via the GitHub UI
+-------------------------------------------
+
+1. Click the pencil icon to edit this file in your fork of the www repo.
```

Comment:
Looking at this again, steps 4 & 5 from what used to be called the prerequisites are kind of necessary here. Sorry I was so focused on not forcing people to install Git when using only the UI. 🙈 
```suggestion
1. In the [Write the Docs www project](https://github.com/writethedocs/www), find the file name corresponding to an [existing issue](https://github.com/writethedocs/www/issues) or page you want to improve.

   For example, /docs/documentarians.rst creates https://www.writethedocs.org/documentarians/.

1. Review formatting guidelines for the Markup style the file uses. The example file above uses [reStructuredText (.rst)](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html).
1. Click the pencil icon to create a fork of the `www` repo and edit the file there.
```

Not a blocker, but might be nice 🤷

Reply:
I included similar changes in my latest commit.